packages/block-editor/src/components/inserter-list-item/index.js

```diff
@@ -45,6 +45,7 @@ function InserterListItem( {
 					onClick();
 				} }
 				disabled={ isDisabled }
+				__experimentalIsFocusable
```

Comment:
Yes, this prop is great. Props to @diegohaz. We should make it stable as we use it in several places or should be doing that, stuff like undo or redo 😃

Reply:
Yeah! I just wonder if it's easy enough to understand with this name (`isFocusable`). I've seen a lot of confusion around the similar `focusable` prop on Reakit, and I'm considering renaming it to `tabbableWhenDisabled` or something like this, which better describes the behavior (it's only focusable with the keyboard, and this behavior only makes sense when the element is disabled).

So, I think we should consider renaming it too when we make it stable.

Reply:
Interesting observation regarding the name. Yes, I'm open to other options that better express the intent 👍🏻

Reply:
Yep, agree that it's not the clearest name as it is.